### PR TITLE
Create a topic about the Adobe Privacy JavaScript Library

### PR DESCRIPTION
**Pull request by jcalcaben**
## Purpose of this pull request

Creates a topic about the Adobe Privacy JavaScript Library

## Affected DevDocs pages

- https://devdocs.magento.com/compliance/privacy/adobe-privacy-js-library.html

### CODE REVIEW

File: src/compliance/privacy/adobe-privacy-js-library.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+Magento and Adobe's data tracking services can store private information applicable to privacy regulations such as the [General Data Protection Regulation (GDPR)][] and [California Consumer Privacy Act (CCPA)][].
+This library provides a unified set of functions for creating privacy-data requests, sending them to each product's implementations, and gathering the responses.
+Use this library to retrieve and remove the data stored in the browser by these data tracking services.
+
+## Installation
+
+Use one of the following methods to get a copy of the library file:
+
+-  Using npm: `npm install @adobe/adobe-privacy`
+-  Download the file from <https://github.com/Adobe-Marketing-Cloud/adobe-privacy>
```
Comment (erikmarr) [inline src/compliance/privacy/adobe-privacy-js-library.md:17]:
```suggestion
-  GitHub: <https://github.com/Adobe-Marketing-Cloud/adobe-privacy>
```

File: src/compliance/privacy/adobe-privacy-js-library.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+: Removes identities from the browser and returns an array of identity objects with a `isDeleteClientSide` boolean property to indicate whether the data has been deleted.
+
+`retrieveThenRemoveIdentities()`
+: This function is similar to `removeIdentities()` in that it retrieves an array of identities and removes them from the browser.
+
+For more information and examples for using these functions, see the [official library documentation][Adobe Privacy JavaScript Library].
+
+### Initialization
+
+Instantiate a new `AdobePrivacy` object to use the Privacy JS Library in your implementation code.
```
Comment (erikmarr) [inline src/compliance/privacy/adobe-privacy-js-library.md:39]:
```suggestion
Instantiate a new `AdobePrivacy` object to use the AdobePrivacy JS Library in your implementation code.
```

**Comment by dobooth** [comment]
running tests

**Comment by hguthrie** [comment]
@erikmarr The hand of fate has been dealt. You must add a section for `whatsnew` or change your labeling strategy.

**Pull request merged**